Extension/tslint.json

```diff
@@ -1,31 +1,184 @@
 {
     "rules": {
+        "adjacent-overload-signatures": true,
+        "align": true,
+        "array-type": false,
+        "arrow-parens": false,
+        "arrow-return-shorthand": true,
+        "await-promise": false,
+        "ban-comma-operator": true,
+        "binary-expression-operand-order": true,
+        "callable-types": true,
+        "class-name": true,
+        "comment-format": true,
+        "completed-docs": false,
+        "no-console": false,
         "curly": true,
+        "cyclomatic-complexity": false,
+        "deprecation": false,
```

Comment:
This is scary. We should know when things are deprecated. I would not mark this as false to ignore.